te/TE_WRAP.py

```diff
@@ -394,7 +394,10 @@ def __buildDocker(type, image_id=None):
         logging.getLogger("paramiko").setLevel(logging.ERROR)
         ssh = paramiko.SSHClient()
         ssh.set_missing_host_key_policy(paramiko.AutoAddPolicy())
-        ssh.connect(host, username=uname, password=pwd)
+        if pwd:
```

Comment:
where ever password is missing we try to authenticate using system default certs. Auto taken care by paramiko / parallelssh libraries.

In future when we add cert based auth, we can change the priority